README.md

```diff
@@ -1,22 +1,23 @@
-# byoLAD
+<img src="byo_LAD.png"  width="200" height="200">
+
+# byoLAD: Your code improvement buddy
 
 byoLAD augments your development experience by reviewing your code, identifying errors, adding documentation, and suggesting general improvements.  Since byoLAD is open-source, you can modify prompts and other configurations to your liking. 
 
-In addition, byoLAD enables you to use your own LLM API, so you can use your favorite AI and control how much you spend. We don't add any extra costs, and there are no extra parties looking at your data. You only have to manage your account with your LLM provider. 
+In addition, byoLAD enables you to use your own Large Language Model (LLM) API, so you can use your favorite AI and control how much you spend. We don't add any extra costs, and there are no extra parties looking at your data. You only have to manage your account with your LLM provider. 
 #### LLM Providers Currently Supported: 
-- OpenAI
-- Google
-
+- [OpenAI](https://openai.com/blog/openai-api)
+- [Google](https://developers.generativeai.google/)
+  
```

Comment:
Let's add sub-bullets for the models and API token instruction pages for each provider.